src/features/Chat/components/Menu/FolderLink.tsx

```diff
@@ -0,0 +1,79 @@
+import { useAppDispatch } from '@/store';
+import { setActiveFolder, setShowFolders } from '@/features/Chat/chatSlice';
+import { useTranslation } from 'react-i18next';
+
+import type { ChatFolder } from '@/features/Chat/chatPageApi';
+
+import styled, { css } from 'styled-components';
+import BackArrowIcon from '@/static/icons/back-arrow.svg';
+import { palette } from '@/components/variables';
+import Text from '@/components/Text';
+
+type Props = {
+  targetFolder: ChatFolder;
+};
+
+const FolderLink = ({ targetFolder }: Props) => {
+  const { t } = useTranslation('chat');
+
+  const dispatch = useAppDispatch();
+
+  const changeActiveFolder = (folder: ChatFolder) => {
+    dispatch(setActiveFolder(folder));
+  };
+
+  const hideFolders = () => {
+    dispatch(setShowFolders(false));
+  };
+
+  const backToActive = targetFolder === 'ok';
```

Comment:
Here again I would change the boolean variable name to **isActiveTabActive** or something

Reply:
Fixed!